services/workflows-service/src/transaction/dtos/get-transactions.dto.ts

```diff
@@ -25,8 +27,8 @@ export class GetTransactionsDto {
   endDate?: Date;
 
   @IsOptional()
-  @IsEnum(['minutes', 'hours', 'days', 'months', 'years'])
-  timeUnit?: 'minutes' | 'hours' | 'days' | 'months' | 'years';
+  @IsEnum(Object.values(TIME_UNITS))
```

Comment:
Do we have a task that checks validation on this field ?

Reply:
Do you mean a test? We don't use.

Reply:
i add some test on bad request once i fixed the response but it might be good to add this check as well